### PR TITLE
Feature: ad performance channels

### PR DESCRIPTION
**Pull request by axelazaid**
add `channel` field in `fct_ad_performance` model

### CODE REVIEW

File: models/marts/advertising/adwords_ad_performance_adapter.sql [hunk 1]
```diff
@@ -13,6 +13,7 @@ joined as (
         criteria_id,
         ad_group_name,
         campaign_id,
+        'google' as utm_source,
```
Comment (eogilvy12) [inline models/marts/advertising/adwords_ad_performance_adapter.sql:16]: should `cpc` be added as the `utm_medium`?

File: models/marts/advertising/adwords_ad_performance_adapter.sql [hunk 2]
```diff
@@ -13,6 +13,7 @@ joined as (
...
         lower(campaign_name) as adwords_campaign_name,
```
Comment (eogilvy12) [inline models/marts/advertising/adwords_ad_performance_adapter.sql:18]: should this `campaign_name` be the `utm_campaign`?

File: packages.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ packages:
     version: 0.7.2
```
Comment (eogilvy12) [inline packages.yml:6]: I think snowplow just had a number of new updates pushed as well if you could update!

**Review by eogilvy12 — CHANGES_REQUESTED** [review]
a few small questions/changes. let me know if you want to chat IRL

File: models/marts/advertising/adwords_ad_performance_adapter.sql (hunk 2 above)
Comment (axelazaid) [reply models/marts/advertising/adwords_ad_performance_adapter.sql:18]: the criteria performance model doesn't have a `utm_campaign` field, just `campaign_name`

Comment (axelazaid) [reply models/marts/advertising/adwords_ad_performance_adapter.sql:18]: the `campaign_name` doesn't match necessarily to the `marketing_campaign`

Comment (eogilvy12) [reply models/marts/advertising/adwords_ad_performance_adapter.sql:18]: gotcha

**Review by eogilvy12 — APPROVED** [review]
if tests are passing this looks good to me

**Pull request merged**